{{cookiecutter.project_slug}}/doc-requirements.txt

```diff
@@ -0,0 +1,2 @@
+sqlalchemy
+sphinx_bootstrap_theme
```

Comment:
Thanks for your PR, very appreciated.

I would suggest to move those requirements to the ``requirements.test.txt`` file as long we are [building documentation with the test config](https://github.com/AnyBlok/cookiecutter-anyblok-project/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/Makefile#L57)

Also I'm wondering why slqalchemy is especially required for build the documentation ? you could make your requirement file depends to an other as we do with [dev requirements file](https://github.com/AnyBlok/cookiecutter-anyblok-project/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/requirements.dev.txt).

Anyway, is that fine if you move that to that file: [requirements.test.txt](https://github.com/AnyBlok/cookiecutter-anyblok-project/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/requirements.test.txt) so you will needs to do the following steps to build the documentation:

```
make setup-tests
make documentation
```

Actually, I'm wondering what require sphinx_bootstrap_theme in the current documentation, should it be added in the doc/conf.py?

Reply:
About bootstrap, it's not a requirements, but we often use it. So i will add it to conf.py also.

About sql-alchemy, it's seems we need it to have autodoc documentation.

Ok for move all to requirements.test.txt.

@jssuzanne say me (in off) to also add CODE.rst and README.rst in autogenerated blok.

Reply:
* **Bootstrap** is a theme that I have often used, If think it hasn't to be add in the template, each cookiecutter user would use their own templating library.
* The **sqlalchemy** import is needed by reathedocs, I don't know why, but I have to add this requirement in all the readthedocs.
* **CODE.rst** is not required by anyblok_doc method, I used it in each blok of anyblok for the audodoc. I wouldn't add it.
* **README.rst** is called if exist in the anyblok_doc. But not only. The System.Blok have a field function to return it. **anyblok_furetui** convert it to html to display it. This file represents the long description of the Blok, the doc string in blok class is the short description.